examples/cucumber/.sauce/config.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1alpha
+kind: espresso
+sauce:
+  region: us-west-1
+  concurrency: 10 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Cucumber and Espresso Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+espresso:
+  app: ./apps/espresso-cucumber.apk
```

Comment:
There is no `./apps/espresso-cucumber.apk` under /examples/cucumber 🤔 
How does the example run? 🤔